serializers/postgres.rb

```diff
@@ -20,7 +20,8 @@ def self.serialize_internal(pg, options = {})
       base.merge!(
         connection_string: pg.connection_string,
         primary: pg.representative_server&.primary?,
-        firewall_rules: Serializers::PostgresFirewallRule.serialize(pg.firewall_rules.sort_by { |fwr| fwr.cidr.version && fwr.cidr.to_s })
+        firewall_rules: Serializers::PostgresFirewallRule.serialize(pg.firewall_rules.sort_by { |fwr| fwr.cidr.version && fwr.cidr.to_s }),
+        metric_destinations: pg.metric_destinations.map { {id: _1.ubid, url: _1.url} }
```

Comment:
Should we add a serializer for it?

Reply:
Since this is very small modification, I prefer to not add a serializer for it. If it becomes more complex, or needs to be used in multiple places in the future, I'd be fine to make this a serializer.